server/pages/home.js

```diff
@@ -45,6 +71,22 @@ function candidate([name, data]) {
       ),
       data.suspended &&
         h('p', { style: 'margin: 0;' }, 'Suspended on:', data.suspended)
+    ),
+    h(
+      'div',
+      { class: 'candidate-overview' },
+      h('h3', { style: 'margin: 0; padding: 0;' }, 'Delegates'),
+      h('ul', { class: 'delegates' },
+        // @TODO: Remove these manual entries once we sort out how to load them dynamically
+        data.delegates[1] && h('li', { class: 'delegate' }, `${nameToHuman(data.delegates[1].state)}: ${data.delegates[1].count}`),
+        data.delegates[0] && h('li', { class: 'delegate' }, `${nameToHuman(data.delegates[0].state)}: ${data.delegates[0].count}`),
+
+        // @TODO: Figure out why this does not work.  Logs correctly, but does not render
+        data.delegates.forEach(delegate => {
```

Comment:
`h` is expecting a list of children components, so if you switch the `forEach` to a `map` and spread the return value you'll get the result your seeking.

```suggestion
        ...data.delegates.map(delegate => {
```